!

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Type=Application
+Terminal=false
+Icon=/home/"username"/.config/galaxy.png
```

Comment:
Wouldn't it be possible to use `~` instead of `/home/"username"`?

Reply:
sure but it some times doesn't work on specfic distro